src/main/java/com/portatlas/Directory.java

```diff
@@ -0,0 +1,37 @@
+package com.portatlas;
+
+import java.io.File;
+import java.util.ArrayList;
+
+public class Directory {
+    public String pathName;
+    public ArrayList files = new ArrayList();
```

Comment:
You should specify the type of List here as `List<String>`.
1. Its good to use the `List` interface so that you can change implementations without changing a bunch of your code.
2. You should also specify the type of content the `List` contains using generics, which is what `<String>` specifies.